### DISCUSSION TITLE
How to resume an unfinished conversion?

### DISCUSSION THREAD

**Discussion by covermeyer24** [nagadomi/nunif#208] [Q&A] [answered]
I see from this [comment](https://github.com/nagadomi/nunif/discussions/129#discussioncomment-9872214), there is a way to resume an unfinished conversion.  I'm not sure how to get it to resume, the checkbox for resume is checked in the GUI but after clicking start, the conversion just starts from the beginning.

**Comment by nagadomi** [comment, marked as answer]
resume for video conversion is not supported. 
resume is only supported for Export and directory input(skip completed videos/images).

**Reply by covermeyer24** [reply]
I see, thanks for the answer.  Is there a chance for video conversion resume to be added as a feature later, or is it not possible?

**Reply by nagadomi** [reply]
It is difficult to implement without extra disk space and overhead.
You can resume the video conversion with [Export -> YML input steps](https://github.com/nagadomi/nunif/issues/97#issuecomment-2027349722), but it will take much extra disk space and processing time.

**Comment by nagadomi** [comment]
Another method is to split the input video in advance and merge it later.
https://video.stackexchange.com/questions/32297/resuming-a-partially-completed-encode-with-ffmpeg
https://video.stackexchange.com/questions/17061/is-there-a-way-to-pause-and-resume-ffmpeg-encoding/17062#17062
If this method is possible with any codec without causing seams and without re-encoding, it could be implemented in software.

Also, if the application is not terminated, `Suspend`(Pause) button is already implemented. https://github.com/nagadomi/nunif/discussions/178
